exercises/binary-search/README.md

```diff
@@ -0,0 +1,113 @@
+# Binary Search
+
+Write a program that implements a binary search algorithm.
```

Comment:
By https://github.com/exercism/problem-specifications/issues/321, we are removing the "Write a program"

Reply:
So the whole sentence should be removed?

Reply:
Just change it to "Implement a binary search algorithm."

Reply:
> So the whole sentence should be removed?

I should have been more clear, therefore I apologise for not being so.

My goal is that the README given in this PR is exactly the README generated by `configlet generate` (documented at https://github.com/exercism/docs/blob/master/maintaining-a-track/regenerating-exercise-readmes.md), with no modifications. Because of time, the track maintainers are unable to maintain any README that is different from the generated README.

This sentence containing "Write a program" is different from the first sentence of https://github.com/exercism/problem-specifications/blob/master/exercises/binary-search/description.md, therefore it is different from the generated README.

I will be generating the README, so you do not need to fix it yourself, but I need you to know it for future PRs.